salt/apps/odlvideo/post_deploy.sls

```diff
@@ -0,0 +1,18 @@
+{% set app_dir = '/opt/{0}'.format(salt.pillar.get('django:app_name')) %}
+
+build_static_assets_for_odlvideo:
+  cmd.script:
+    - name: {{ app_dir }}/webpack_if_prod.sh
+    - env:
+        - NODE_ENV: production
```

Comment:
Do we want to use the env variable here or will it always be `production`?

Reply:
This will always be `production` because it is for controlling whether the webpack build gets executed. Here `NODE` is referring to node.js